File: app/src/main/java/com/arejas/dashboardofthings/presentation/ui/helpers/AddEditSensorActuatorPresenter.java
package com.arejas.dashboardofthings.presentation.ui.helpers;

public interface AddEditSensorActuatorPresenter {

    public void pickImage();

    public void cancelImagePicked();

    public void pickLocation();

    public void cancelLocationPicked();

    public void addHttpHeader();

    public void cancelHttpHeader(String headerName);

    public void networkSelected(int ordinal);

    public void messageTypeSelected(int ordinal);

    public void dataTypeSelected(int ordinal);

}

File: app/src/main/java/com/arejas/dashboardofthings/presentation/ui/helpers/AddEditSensorActuatorPresenter.kt
package com.arejas.dashboardofthings.presentation.ui.helpers

interface AddEditSensorActuatorPresenter {

    fun pickImage()

    fun cancelImagePicked()

    fun pickLocation()

    fun cancelLocationPicked()

    fun addHttpHeader()

    fun cancelHttpHeader(headerName: String)

    fun networkSelected(ordinal: Int)

    fun messageTypeSelected(ordinal: Int)

    fun dataTypeSelected(ordinal: Int)

}
